recipe/meta.yaml

```diff
@@ -28,7 +29,7 @@ requirements:
     - numpy 1.19.*                           # [build_platform != target_platform]
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - bazel
+    - bazel 3        # [not (osx and arm64)]
```

Comment:
I would still like to get rid of this pinning again. Can you try without it and post the error?

Reply:
That is fair. But I would really like to get help from the grpcio maintainers first. Its kinda painful to try on my local machine (especially during the weekday).

Reply:
Seems like it is kinda working
```
    bazel:                    4.2.1-h4d98ba0_1       conda-forge
```